test/scripts/validation/scenario_config2

```diff
@@ -0,0 +1,7 @@
+---
+submariner: true
+nodes: control-plane worker
+clusters:
+  cluster1:
```

Comment:
```suggestion
  clusters-test:
```

Perhaps something like this, in case anyone has any kind clusters already deployed with the name `cluster1` on their environment.